lostmediafinder/finder.py

```diff
@@ -116,12 +122,12 @@ async def _run(cls, id, session: aiohttp.ClientSession, includeRaw=True) -> T:
         results.sort(key=lambda x: x[1], reverse=True)
         quality_order = [
             'maxresdefault.jpg',
+            'sddefault.jpg',
```

Comment:
Is there a source for the order of what is higher quality than what? Might be helpful in the future.

(The name also suggests to me that it isn't the highest quality (SD vs HD), but I really have no idea.)